src/olympia/files/admin.py

```diff
@@ -1,10 +1,78 @@
 from django.contrib import admin
+from django.utils.html import format_html
+from django.conf.urls import url
+from django.shortcuts import get_object_or_404
+
+from olympia.versions.views import download_file
+from olympia.addons.models import Addon
+from olympia.amo.urlresolvers import reverse
 
 from .models import File
 
 
+@admin.register(File)
 class FileAdmin(admin.ModelAdmin):
+    view_on_site = False
+
     raw_id_fields = ('version',)
+    list_display = (
+        '__unicode__', 'addon_slug', 'addon_guid')
+    search_fields = (
+        '^version__addon__guid',
+        '^version__addon__slug',)
```

Comment:
It's unfortunate we can't do exact matches because I expect the users of this will know the exact addon they're looking for.

Reply:
We can do a [iexact](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#std:fieldlookup-iexact) match, which may make sense for the guid, I'm not too sure for the slug :thinking: 

Could be part of a future improvement though, I just wanted to add the ability of filtering the list down for easier access.